### ISSUE TITLE
Working branch for submitting Swift pods and extensions to the App Store?

### ISSUE THREAD

**Issue by basememara** [CocoaPods/CocoaPods#3721]
There seems to be multiple issues regarding archiving apps with extensions and Swift framework pods. Mainly, I hoping branch "mr-deduplicate-frameworks" would solve just enough issues to archive to the App Store, but other issues remain that need to be merged in. 

Has anyone found a branch that works to test archiving and submitting to the App Store? I have an app with watch and today extensions that have been stuck in the water for almost a month and would like to provide some test feedback. I tried branch "mr-deduplicate-frameworks" but my app couldn't compile with it.

**Comment by neonichu** [comment]
Could you elaborate more on which issues you had with the "mr-deduplicate-frameworks" branch?

**Comment by basememara** [comment]
Sure, I left in this in the #3550 thread. I'm getting this compile error:
```
PhaseScriptExecution Embed\ Pods\ Frameworks /Users/.../Library/Developer/Xcode/DerivedData/.../ArchiveIntermediates/.../IntermediateBuildFilesPath/....build/Release-iphoneos/...-....sh
    cd "/Users/.../Solutions/Xcode/..."
    /bin/sh -c \"/Users/.../Library/Developer/Xcode/DerivedData/.../Build/Intermediates/ArchiveIntermediates/.../IntermediateBuildFilesPath/....build/Release-iphoneos/....build/Script-....sh\"

mkdir -p /Users/.../Library/Developer/Xcode/DerivedData/.../Build/Intermediates/ArchiveIntermediates/.../BuildProductsPath/Release-iphoneos/....app/Frameworks
rsync -av --filter - CVS/ --filter - .svn/ --filter - .git/ --filter - .hg/ --filter - Headers/ --filter - PrivateHeaders/ --filter - Modules/ /Users/.../Library/Developer/Xcode/DerivedData/.../Build/Intermediates/ArchiveIntermediates/.../BuildProductsPath/Release-iphoneos/Pods/MZTimerLabel.framework /Users/.../Library/Developer/Xcode/DerivedData/.../Build/Intermediates/ArchiveIntermediates/.../BuildProductsPath/Release-iphoneos/....app/Frameworks
building file list ... rsync: link_stat "/Users/.../Library/Developer/Xcode/DerivedData/.../Build/Intermediates/ArchiveIntermediates/.../BuildProductsPath/Release-iphoneos/Pods/MZTimerLabel.framework" failed: No such file or directory (2)
done

sent 29 bytes  received 20 bytes  98.00 bytes/sec
total size is 0  speedup is 0.00
rsync error: some files could not be transferred (code 23) at /SourceCache/rsync/rsync-45/rsync/main.c(992) [sender=2.6.9]
Command /bin/sh failed with exit code 23
```
I was advised to merge #3675 which I attempted but getting tangled up in Ruby woes (didn't know how to create a merged executable and still researching that).

**Comment by basememara** [comment]
Not sure if this has anything to do with it, but my compile error is complaining about the only non-Swift pod in my project. Maybe a clue or coincidence.

**Comment by neonichu** [comment]
OK, the error you're experiencing should indeed be solved by #3675 - I'm afraid there is no better way than manually merging it right now, but we're planning on releasing a new beta version soon :tm:

**Comment by basememara** [comment]
The master branch works now :tada: due to these being merged in: #3550, #3675, #3737. I was able to archive and send it all the way to TestFlight! Thank you, thank you!!! @neonichu @segiddins @mrackwitz

![carlton-dance](https://cloud.githubusercontent.com/assets/892152/8389052/8343e040-1c38-11e5-80f5-b23c0d534d4e.gif)
